nixos/tests/all-tests.nix

```diff
@@ -439,6 +439,7 @@ in {
   fontconfig-default-fonts = handleTest ./fontconfig-default-fonts.nix {};
   forgejo = handleTest ./forgejo.nix { forgejoPackage = pkgs.forgejo; };
   forgejo-lts = handleTest ./forgejo.nix { forgejoPackage = pkgs.forgejo-lts; };
+  foundationdb = handleTest ./foundationdb.nix {};
```

Comment:
Please migrate to runTest https://github.com/NixOS/nixpkgs/issues/386873

Reply:
Done. Unable to run and verify at the moment, but anyways until we fix the underlying foundationdb coredump issue it's a moot point.